scheduler/scheduler-server/src/main/java/org/ow2/proactive/utils/JettyStarter.java

```diff
@@ -269,11 +266,18 @@ private void startServer(Server server, String schedulerHost, int restPort, Stri
             logger.error("Failed to start web applications", e);
             System.exit(3);
         }
+        return null;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Return an empty collection instead of null. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://master.ci.activeeon.com:9000/coding_rules#rule_key=squid%3AS1168)